src/map/skill.cpp

```diff
@@ -5582,7 +5582,11 @@ int32 skill_castend_damage_id (struct block_list* src, struct block_list *bl, ui
 
 	case RG_BACKSTAP:
 		{
-			if (!check_distance_bl(src, bl, 0)) {
+			bool is_skip_check = false;
+			#ifdef RENEWAL
+			is_skip_check = true;
+			#endif
+			if (is_skip_check || !check_distance_bl(src, bl, 0)) {
```

Comment:
```suggestion
#ifndef RENEWAL
			if (check_distance_bl(src, bl, 0))
				break;
#endif
```
I feel like this is a way more simple way to put it.

But you will also need to remove the indentation on the lines below.

That makes it easier to later put it the correct direction code.